examples/top_block.py

```diff
@@ -21,11 +21,11 @@
             print("Warning: failed to XInitThreads()")
 
 from PyQt5 import Qt
-from gnuradio import qtgui
-from gnuradio.filter import firdes
-import sip
+from PyQt5.QtCore import QObject, pyqtSlot
+from gnuradio import analog
 from gnuradio import blocks
 from gnuradio import gr
+from gnuradio.filter import firdes
 from gnuradio.fft import window
 import sys
 import signal
```

Comment:
apparently I cannot add comments outside of the pre-loaded code around the changes, but this class really needs to be updated which I know will trickle down to wherever it is used but it should be named `TopBlock`